google/cloud/documentai_toolbox/wrappers/page.py

```diff
@@ -90,97 +106,184 @@ def sample_table_to_csv():
+        init=True, repr=False
+    )
+    text: str = dataclasses.field(init=True, repr=False)
+    _page: Page = dataclasses.field(init=True, repr=False)
+
+    lines: List[Line] = dataclasses.field(init=False, repr=False)
+
+    def __post_init__(self):
+        self.lines = _get_children_of_element(self.documentai_object, self._page.lines)
 
 
 @dataclasses.dataclass
-class Line:
-    """Represents a wrapped documentai.Document.Page.Line.
+class Block:
+    """Represents a wrapped documentai.Document.Page.Block.
 
     Attributes:
-        documentai_line (google.cloud.documentai.Document.Page.Line):
-            Required. The original google.cloud.documentai.Document.Page.Line object.
+        documentai_object (google.cloud.documentai.Document.Page.Block):
+            Required. The original google.cloud.documentai.Document.Page.Block object.
         text (str):
             Required. UTF-8 encoded text.
     """
 
-    documentai_line: documentai.Document.Page.Line
-    text: str
+    documentai_object: documentai.Document.Page.Block = dataclasses.field(
+        init=True, repr=False
+    )
+    text: str = dataclasses.field(init=True, repr=False)
+    _page: Page = dataclasses.field(init=True, repr=False)
 
+    paragraphs: List[Paragraph] = dataclasses.field(init=False, repr=False)
 
-@dataclasses.dataclass
-class FormField:
-    """Represents a wrapped documentai.Document.Page.FormField.
+    def __post_init__(self):
+        self.paragraphs = _get_children_of_element(
+            self.documentai_object, self._page.paragraphs
+        )
+
+
+def _table_wrapper_from_documentai_table(
+    documentai_table: documentai.Document.Page.Table, text: str
+) -> Table:
+    r"""Returns a Table.
+
+    Args:
+        documentai_table (documentai.Document.Page.Table):
+            Required. A documentai.Document.Page.Table.
+        text (str):
+            Required. UTF-8 encoded text in reading order
+            from the document.
+
+    Returns:
+        Table:
+            A Table.
 
-    Attributes:
-        documentai_formfield (google.cloud.documentai.Document.Page.FormField):
-            Required. The original google.cloud.documentai.Document.Page.FormField object.
-        field_name (str):
-            Required. The form field name
-        field_value (str):
-            Required. The form field value
     """
 
-    documentai_formfield: documentai.Document.Page.FormField
-    field_name: str
-    field_value: str
+    header_rows = _table_rows_from_documentai_table_rows(
+        table_rows=list(documentai_table.header_rows), text=text
+    )
+    body_rows = _table_rows_from_documentai_table_rows(
+        table_rows=list(documentai_table.body_rows), text=text
+    )
+
+    return Table(
+        documentai_table=documentai_table, body_rows=body_rows, header_rows=header_rows
+    )
+
+
+def _get_hocr_bounding_box(
+    element_with_layout: ElementWithLayout,
+    dimension: documentai.Document.Page.Dimension,
```

Comment:
We may not need this any more, since every element_with_layout has access to its containing page?

Reply:
we will still need it since we need to get the bounding boxes, we can instead of calling this in `to_hocr` we can call it at object initialization.

Reply:
Is the page's dimension already included in the element's `_page`?

Reply:
This still stands.  If i understand correctly, `dimension` is a property of `element_with_layout._page` and does not need to be an argument of this function.